napari/plugins/_plugin_manager.py

```diff
@@ -606,7 +606,7 @@ def get_widget(
         Raises
         ------
         KeyError
-            If plugin `plugin_name` does not provide any widgets
+            If plugin `plugin_name` is not installed or does not provide any widgets
```

Comment:
Minor nitpick: might be nice to have a period at the end.

```suggestion
            If plugin `plugin_name` is not installed or does not provide any widgets.
```